playbooks/roles/edxapp/files/update_theme.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT license. See LICENSE file on the project webpage for details.
+
+set -x
+
+sudo cp /oxa/oxa-tools/config/server-vars.yml /edx/app/edx_ansible/
```

Comment:
See if you can get the edx config approach to work, which would remove need for update_theme and playbook task